internal/pkg/utils/testhelper/terminal/terminal_linux.go

```diff
@@ -0,0 +1,240 @@
+//go:build !windows && !darwin
+// +build !windows,!darwin
+
+package terminal
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ActiveState/vt10x"
+	"github.com/Netflix/go-expect"
+	"github.com/acarl005/stripansi"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/testhelper"
+)
+
+// console implements Console.
+type console struct {
+	*expect.Console
+	state *vt10x.State
+	tty   *tty
+}
+
+// tty implements Tty, it is an os.File wrapper for virtual terminal TTY.
+// Unlike os.File, when Close() is called,
+// all running Read and Write operations are immediately terminated,
+// so the test timeout does not occur.
+type tty struct {
+	file   *os.File
+	closed chan struct{}
+}
```

Comment:
Ak nastala v teste situacia, ze sa dany `ExpectString` nenasiel, ... tak to sice skoncilo chybou, ale test sa zasekol.
Problem bol, ze aj ked sa zavolalo `console.Tty().Close()`, tak to nezrusilo prebiehajuce citanie `Read`, ktore je blokujuce (tak funguje Go `*os.File`).

Reply:
Vid implementaciu na konci suboru.